### ISSUE TITLE
Moderation Does Not Support Dark Theme

### ISSUE THREAD

**Issue by IntegerMan** [forem/forem#5738]
Using the dark theme (non black/green, the older dark theme), if I go in to Moderation, I get a very hard to read user experience.

This makes it hard to be an efficient moderator as there are no true other themes beyond the dark themes.

**To Reproduce**

1. Set your theme to the black and teal
1. Be awesome
1. Become a Tag Moderator
1. Go to Moderation in Edge or Chrome on Windows 10

**Expected behavior**

Moderation theming works just as awesomely as the rest of the site.

**Screenshots**

![image](https://user-images.githubusercontent.com/5049957/73148006-0f9e8b80-4088-11ea-8292-67e8ffa37327.png)

![image](https://user-images.githubusercontent.com/5049957/73148069-4d9baf80-4088-11ea-91f7-4a369dc2b268.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Edge / Chrome
- Version:  79.0.309.71 (Edge)

**Comment by catalinpit** [comment]
Hello, can I pick this up, please?

Thank you!

**Comment by rhymes** [comment]
@catalinpit yes, thank you! Let us know if you need some assistance  :-)

**Comment by IntegerMan** [comment]
Thank you so much, @catalinpit, I really appreciate it.

**Comment by ItsASine** [comment]
A workaround I have is to use a Dark Mode extension just to get rid of the bright and teal on the /mod page. This doesn't work for all views and systems, but it at least lets me keep working without needing a white UI.

Particularly [this extension](https://chrome.google.com/webstore/detail/dark-mode/dmghijelimhndkbmpgbldicpogfkceaj) with theme 14, but the theory holds for any overriding dark mode.

**Comment by catalinpit** [comment]
No need for a workaround anymore.

This is how it's going to look: https://github.com/thepracticaldev/dev.to/pull/7085

**Comment by rhymes** [comment]
Closed by https://github.com/thepracticaldev/dev.to/pull/7096

Thanks again @catalinpit, great work! 🔥 

@IntegerMan @ItsASine @OmisNomis @stroudn1 let us know if you're still experiencing the issue :)

**Comment by ItsASine** [comment]
![](https://media.giphy.com/media/qKNfbNbEN5ylW/giphy.gif)

**Comment by michael-tharrington** [comment]
Reopening this as a mod discovered that dates are not visible on dev.to/mod when using dark theme:

![Image 2020-11-30 at 5 23 01 PM](https://user-images.githubusercontent.com/16007075/100673032-c4e94200-3330-11eb-9602-e2d6510ba0e9.png)
